manifests/s/Synology/SurveillanceStationClient/2.2.1-2564/Synology.SurveillanceStationClient.locale.en-US.yaml

```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Synology.SurveillanceStationClient
+PackageVersion: 2.2.1-2564

```

Comment:
```suggestion
PackageVersion: 2.2.1.2564
```

Reply:
Thanks for reviewing the PR, this package version reflects the same naming scheme as the other synology packages already in the repo, such as: 
https://github.com/microsoft/winget-pkgs/blob/master/manifests/s/Synology/NoteStationClient/2.2.4-703/Synology.NoteStationClient.locale.en-US.yaml
I had figured keeping the versioning consistent might be better, however I'm not particularly worried either way. Please let me know if you'd still like to keep your suggested change.